crates/ra_db/src/input.rs

```diff
@@ -159,7 +160,7 @@ pub struct ExternSourceId(pub u32);
 
 #[derive(Default, Debug, Clone, PartialEq, Eq)]
 pub struct Env {
-    entries: FxHashMap<String, String>,
+    entries: FxHashMap<OsString, OsString>,
```

Comment:
Here, I think `String` is actually the right type. The sink for this things is the `env!` macro, and it takes and returns a `str`. If you have to do a conversion, it's better to do it as soon as possible.

Reply:
I can revert that part. I changed it because it seemed to make sense -- at some point we might get an `os_env!` macro, for example.

Reply:
Done.